ext/lib/mgmt/mcumgr/README-zephyr.md

```diff
@@ -0,0 +1,30 @@
+## Building and using mcumgr with Zephyr
+
+### Configuration
+
+The `samples/smp_svr/zephyr/prj.conf` file provides a good starting point for
+configuring an application to use *mcumgr*.  The major configuration settings
+are described below:
+
+| Setting       | Description   | Default |
+| ------------- | ------------- | ------- |
+| `CONFIG_MCUMGR` | Enable the mcumgr management library. | n |
+| `CONFIG_MCUMGR_CMD_FS_MGMT` | Enable mcumgr handlers for file management | n |
+| `CONFIG_MCUMGR_CMD_IMG_MGMT` | Enable mcumgr handlers for image management | n |
+| `CONFIG_MCUMGR_CMD_LOG_MGMT` | Enable mcumgr handlers for log management | n |
+| `CONFIG_MCUMGR_CMD_OS_MGMT` | Enable mcumgr handlers for OS management | n |
+
+### Building
+
+Your application must specify mcumgr as a link-time dependency.  This is done
+by adding the following to your application's `CMakeLists.txt` file:
+
+```
+zephyr_link_libraries(
+    MCUMGR
+)
+```
+
+### Known issues
+
+If the Bluetooth stack runs out of ACL transmit buffers while a large mcumgr response is being sent, the buffers never get freed.  This appears to trigger a net_buf leak in the stack.
```

Comment:
Are you sure that this is a leak, and not a deadlock? Are the responses larger than 23 bytes? If so, it's possible you've been hitting the deadlock that #6110 is trying to fix. Could you check if you still have an issue with that PR included, i.e. try to lower the buffer counts in your configuration.

Reply:
Thanks @jhedberg.  That explanation is much more plausible.  I will try again with the fix and update the doc accordingly.

Reply:
@ccollins476ad not sure if you attempted to test my PR yet, but note that it had a typo which would have caused the new fragment pool to never be used. It's been fixed now, so please use the latest version of #6110

Reply:
#6110 fixed it.  Now all the management commands work with the default `BT_L2CAP_TX_BUF_COUNT` value of 3.